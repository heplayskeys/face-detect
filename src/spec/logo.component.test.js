import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Logo from '../components/logo/logo.component';

describe('<Logo />', () => {
	it('returns correct logo snapshot', () => {
		const wrapper = shallow(<Logo />);
		expect(shallowToJson(wrapper)).toMatchSnapshot();
	});
});
