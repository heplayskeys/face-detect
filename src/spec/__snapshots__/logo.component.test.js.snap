// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> returns correct logo snapshot 1`] = `
<div
  className="face-detect-logo ma4 mt0"
>
  <Tilt
    className="Tilt br2 shadow-2"
    options={
      Object {
        "max": 25,
        "scale": 1.15,
      }
    }
    style={
      Object {
        "height": 150,
        "width": 150,
      }
    }
  >
    <div
      className="Tilt-inner pa3"
    >
      <img
        alt="Logo"
        src="face.png"
      />
      <div>
        face grabber
      </div>
    </div>
  </Tilt>
</div>
`;
